src/weakref.cc

```diff
@@ -31,9 +31,8 @@ class proxy_container {
 
 
 Nan::Persistent<ObjectTemplate> proxyClass;
-
 Nan::Callback *globalCallback;
-
+Nan::HandleScope scope;
```

Comment:
I'm not so sure that having a global HandleScope like this is a good idea… Probably better that we find the entry point where this code is failing and add the scope there instead.